inc/experiments/namespace.php

```diff
@@ -542,10 +542,11 @@ function get_ab_test_traffic_percentage_for_post( string $test_id, int $post_id
  *
  * @param string $test_id The test ID.
  * @param int $post_id Post ID to get test data for.
- * @return float[] Array of percentages.
+ * @return array $traffic_percent array
```

Comment:
This bit was fine fwiw - because it's in a doc block it won't affect how the code runs but can give editors a bit more of an idea of what the function should return. I think this kind of thing is possible in PHP8, so you could do type declarations like `float[]` meaning an array of floating point numbers.

Reply:
Thank you Rob and noted, Should I restore the doc block changes?